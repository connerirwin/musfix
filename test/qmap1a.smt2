; Constraints can generate multiple variants
(qualif Less ((a int)(b int)) (< a b))

(wf $k0 ((v0 int)(v1 int)))
