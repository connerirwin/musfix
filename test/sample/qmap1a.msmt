; Constraints can generate multiple variants
(qualif Less ((a Int)(b Int)) (< a b))

(wf $k0 ((v0 Int)(v1 Int)))
