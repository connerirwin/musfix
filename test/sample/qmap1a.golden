
Preparing to find greatest fixpoint...

Inputs
--------
Reading from file test/sample/qmap1a.msmt
Qualifier "Less" [Var IntS "a",Var IntS "b"] (Binary Lt (Var IntS "a") (Var IntS "b"))
WFConstraint "$k0" [Var IntS "v0",Var IntS "v1"]

QMAP
--------
fromList [("$k0",QSpace {_qualifiers = [Binary Lt (Var IntS "v0") (Var IntS "v1"),Binary Lt (Var IntS "v1") (Var IntS "v0")], _maxCount = 2})]

Candidates
--------
[Candidate {solution = fromList [("$k0",fromList [])], validConstraints = fromList [], invalidConstraints = fromList [], label = "0"}]


Final candidates: 
0: [$k0 -> {}] (0 0)

